### ISSUE TITLE
Improvement: before_standard_html_head: ease child theming by checking for boost_union parentage

### ISSUE THREAD

**Issue by danowar2k** [moodle-an-hochschulen/moodle-theme_boost_union#245]
before_standard_html_head currently checks if the currently selected theme is boost_union and doesn't do anything if it isn't.

To reuse the Font Awesome setting and the flavours a child theme currently has to at least copy the function and change it to its own name.

It would probably be child theming friendlier if boost_union checked if the current theme was boost_union or had boost_union as one of its parents. That way, the function wouldn't have to be reimplemented.

This checking could also be a setting ("ancestry allowed" or something), but that is for another time ;-)

I'll submit a PR for what I mean...

**Comment by danowar2k** [comment]
A question while I am doing the PR: Why the condition 
```
$CFG->theme = 'boost_union'
```
and not
```
$PAGE->theme->name == 'boost_union'
```

Meaning, why is it important that the globally configured theme is boost_union and not just theme on this page?

**Comment by abias** [comment]
This was solved in #246
